tests/unit_tests/conftest.py

```diff
@@ -110,7 +110,7 @@ def add_players(gameobj: BaseGame, n: int, team: int=None):
     current = len(game.players)
     players = []
     for i in range(current, current+n):
-        players.append(Player(id=i+1, login='Player '+str(i+1), global_rating=(1500, 500)))
+        players.append(Player(player_id=i + 1, login='Player ' + str(i + 1), global_rating=(1500, 500)))
```

Comment:
Looks good! Glad to use some of the newer Python functionality - I'm a big fan of using F-strings where possible instead of %-formatting and str.format(),